torchgeo/transforms/indices.py

```diff
@@ -147,21 +147,50 @@ def __init__(self, index_red: int, index_nir: int) -> None:
             index_nir: index of the Near Infrared (NIR) band in the image
         """
         super().__init__(index_a=index_red, index_b=index_nir)
-
-
+        
+        
 class AppendNDWI(AppendNormalizedDifferenceIndex):
     """Normalized Difference Water Index (NDWI).
-
     If you use this index in your research, please cite the following paper:
-
     * https://doi.org/10.1080/01431169608948714
     """
 
     def __init__(self, index_green: int, index_nir: int) -> None:
         """Initialize a new transform instance.
-
         Args:
             index_green: index of the Green band in the image
             index_nir: index of the Near Infrared (NIR) band in the image
         """
         super().__init__(index_a=index_green, index_b=index_nir)
+
+        
+class AppendSWI(AppendNormalizedDifferenceIndex):
+    """Standardized Water-Level Index (SWI) .
+
+    If you use this index in your research, please cite the following paper:
+    * https://www.mdpi.com/2073-4441/13/12/1647/html"""
```

Comment:
Can you switch these links to use DOIs? E.g. https://doi.org/10.3390/w13121647

Reply:
done

Reply:
i have closed the 2 PR this is the one with the required changes